src/main/java/com/neuvector/Scanner.java

```diff
@@ -185,16 +175,16 @@ private static String pullDockerImage(NVScanner nvScanner) {
 
         if(!nvRegistryUser.equals("") && !nvRegistryPassword.equals("")){
             String[] cmdArgsDockerLogin = {"docker", "login", "-u", nvRegistryUser, "-p", nvRegistryPassword, nvRegistryURL};
-            errorMessage = runCMD(cmdArgsDockerLogin, log);
+            errorMessage = runCMD(cmdArgsDockerLogin, log, null);
```

Comment:
New parameter is the output capturer. We do not need the output captured for `docker login`.

Reply:
When scanning against nexus private registry, we need to capture docker login / logout for troubleshooting purpose. So please do not remove this. 

[INFO] Invoking local scan for: container:nginx
[INFO] Login Succeeded!

Reply:
We still have these logs. What I meant is we do not need them in nexus IQ cli.